packages/cactus-plugin-odap-hermes/src/main/typescript/public-api.d.ts

```diff
@@ -0,0 +1 @@
+export * from "./generated/openapi/typescript-axios/api";
```

Comment:
@jscode017 What is the purpose of this file? Seems redundant.

Reply:
Sure, have deleted it.